src/main/java/io/github/cottonmc/resources/block/DefaultBlocks.java

```diff
@@ -0,0 +1,22 @@
+package io.github.cottonmc.resources.block;
+
+import io.github.cottonmc.cotton.registry.CommonBlocks;
+import net.minecraft.block.*;
+import net.minecraft.block.Block.*;
+
+public class DefaultBlocks {
+
+    /**Registers an Item based on its name if it is supported or returns it if it already exists.
+    * @param name The name of the block that shall be registered.
+     * @return The Block associated with the given name or null if no Block for that name was found
+     */
+    public static Block provide(String name){
+        switch(name){
+            case "copper_ore":
+                return CommonBlocks.register(name, new OreBlock(Settings.of(Material.STONE).strength(3.0F, 3.0F)));
```

Comment:
maybe okay for now, but this feels like it should be data somewhere, like a json file?

Reply:
yeah, writing something out as a huge switch block seems like a bad idea.

Reply:
It's better than an if-else chain. Switching to a data driven model is something we can consider in the future, but for now this should do the trick.